### PR TITLE
Error if setting is passed too many parameters

### PR DESCRIPTION
**Pull request by Minoru**
This is inspired by https://github.com/newsboat/newsboat/issues/1568, which was caused by Newsboat silently ignoring "%u" in the following config line:

    browser xdg-open %u

When the user later added &, it was silently ignored too, confusing the user.

Reviews are welcome as always! I'll merge this in three days.

### CODE REVIEW

File: src/confighandlerexception.cpp [hunk 1]
```diff
@@ -17,13 +19,17 @@ const char* ConfigHandlerException::get_errmsg(ActionHandlerStatus status)
...
+	case ActionHandlerStatus::TOO_MANY_PARAMS:
+		return _("too many parameters.");
 	case ActionHandlerStatus::INVALID_COMMAND:
 		return _("unknown command (bug).");
 	case ActionHandlerStatus::FILENOTFOUND:
 		return _("file couldn't be opened.");
-	default:
-		return _("unknown error (bug).");
+	case ActionHandlerStatus::VALID:
+		assert(0 && "unreachable, because this function is only called on errors");
```
Comment (dennisschagt) [inline src/confighandlerexception.cpp:29]: It looks like the `VALID` option is never used, so I think it is better to just remove it from `configparser.h`.

**Review by dennisschagt — APPROVED** [review]
Looks good!

File: src/confighandlerexception.cpp (hunk 1 above)
Comment (Minoru) [reply src/confighandlerexception.cpp]: Right; thanks! Fixed in 7efb07a.

**Comment by Minoru** [comment]
Since you approved the PR and addressing your only comment was trivial, I'm going to enable auto-merge here.

**Pull request merged**